src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -25,11 +26,18 @@ const VerifyEmailAddress = React.memo(function VerifyEmailAddress({
   const theme = useTheme()
   const [emailAddress, setEmailAddress] = useState('')
   const [emailInvalid, setEmailInvalid] = useState(false)
+  const [errorMessage, setErrorMessage] = useState('')
```

Comment:
Perhaps we could just use a single state for the email error? E.g. if there's an error, it's obviously invalid :).